apps/ui/tailwind.config.ts

```diff
@@ -98,7 +98,7 @@ export default {
       'electron-titlebar-height': `${ELECTRON_TITLEBAR_HEIGHT}px`,
       'header-height-with-offset': `${TOTAL_NAV_HEIGHT - 1}px`,
       'header-with-section-height': `${TOTAL_WITH_SECTION}px`,
-      'header-with-section-height-with-offset': `${TOTAL_WITH_SECTION + 1}px`
+      'header-with-section-height-with-offset': `${TOTAL_WITH_SECTION - 1}px`
```

Comment:
Do you guys know why do we use those offsets? I removed those and it works fine for me on both iOS and desktop.

@wa0x6e @ChaituVR